### DISCUSSION TITLE
Source map error: Error: JSON.parse Source Map URL: livewire.js.map

### DISCUSSION THREAD

**Discussion by sscotti** [livewire/livewire#3260] [7. Bugs (archived)]
Not sure if this is a bug or just an annoying warning that I keep seeing in my console.

That seems to get triggered by:  //# sourceMappingURL=livewire.js.map in the last line of the .js for livewire, livewire/livewire.js?id=e6704f81026a73a52725 on my system.

Source map error: Error: JSON.parse: end of data after property value in object at line 2 column 404276 of the JSON data
Resource URL: https://orthanc.test/livewire/livewire.js?id=e6704f81026a73a52725
Source Map URL: livewire.js.map

**Comment by joshhanley** [comment]
@sscotti which version of Livewire are you using?

In your terminal run `composer show -D` to see which exact patch version you are using.

**Reply by sscotti** [reply]
Not sure that it depends on the version.  I just set it to 2.5.1 because of that other bug and it still shows the sourcemap warning in the debugger.  I have been seeing that for awhile.  It doesn't seem to break anything, just an annoying warning.

This is what is at the end of the JS file.  Looks like it is just trying to parse that as JSON and the commented line messes things up.

. . . .    olling(),dispatch("livewire:available"),Livewire}));
//# sourceMappingURL=livewire.js.map

krlove/eloquent-model-generator       1.3.7    Eloquent Model Generator
laracademy/generators                 3.2      This package will generate a Laravel Model based on your database table itself, filling in the requ...
laracasts/generators                  2.0.0    Advanced Laravel generators, that include schema information.
laravel/framework                     v8.49.2  The Laravel Framework.
laravel/jetstream                     v2.3.11  Tailwind scaffolding for the Laravel framework.
laravel/sanctum                       v2.11.2  Laravel Sanctum provides a featherweight authentication system for SPAs and simple APIs.
laravel/tinker                        v2.6.1   Powerful REPL for the Laravel framework.
league/oauth2-google                  3.0.4    Google OAuth 2.0 Client Provider for The PHP League OAuth2-Client
**livewire/livewire                     v2.5.1   A front-end framework for Laravel.**

**Reply by joshhanley** [reply]
@sscotti hmm yeah I remember having that issue too. But I don't now. I'm not sure what I did haha to remove it. Will have a dig!

**Reply by sarmenb** [reply]
what gets rid of this problem is by doing php artisan vendor:publish then choose livewire (assets)

**Reply by sscotti** [reply]
Thanks.  I'll try that.  Do I need to do this also:

"If you would like to make sure your public assets are always up-to-date, you can add this command to the post-update-cmd list in your composer.json file.".  It mentions that for older versions of Laravel.  Just want to keep things UTD automatically if possible.
